### ISSUE TITLE
Problem in Publish with append in trunk 4313

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mondain/red5#178]
```
What steps will reproduce the problem?
1. Publish a stream in "LIVE" application with "append"
2. First time the file is write, but second time gives erros and de file is not 
append with any data

What is the expected output? What do you see instead?
The file have to be appended with new data. Bellow the error.

What version of the product are you using? On what operating system?
Trunk 4313. Linux Fedora-16

Please provide any additional information below.
Attach the error and RED5 log with debug enable
```

Original issue reported on code.google.com by `activep...@gmail.com` on 5 Feb 2012 at 10:49
- Blocking: #265
- Blocked on: #148
- Merged into: #148

Attachments:
- [red5.log](https://storage.googleapis.com/google-code-attachments/red5/issue-178/comment-0/red5.log)
- [error.log](https://storage.googleapis.com/google-code-attachments/red5/issue-178/comment-0/error.log)
